docs/data/migration/migration-data-grid-v5/migration-data-grid-v5.md

```diff
@@ -13,6 +13,28 @@ In `package.json`, change the version of the data grid package to `next`.
 
 Using `next` ensures that it will always use the latest v6 alpha release, but you can also use a fixed version, like `6.0.0-alpha.0`.
 
+## Run codemods
+
+The `preset-safe` codemod will automatically adjust the bulk of your code to account for breaking changes in v6. You can run `v6.0.0/data-grid/preset-safe` targeting only Data Grid or `v6.0.0/preset-safe` to target Date and Time pickers as well.
+It should be only applied **once per folder.**
+
+```sh
+// Data Grid specific
+npx @mui/x-codemod v6.0.0/data-grid/preset-safe <path>
+// Target Date and Time pickers as well
+npx @mui/x-codemod v6.0.0/preset-safe <path>
+```
+
+:::info
+If you want to run the transformers one by one, check out the transformers included in the [preset-safe codemod for data grid](https://github.com/mui/mui-x/blob/next/packages/x-codemod/README.md#preset-safe-for-data-grid) for more details.
+:::
+
+Breaking changes that are handled by this codemod are denoted by a ✅ emoji in the table of contents on the right side of the screen.
+
+If you have already applied the `v6.0.0/data-grid/preset-safe` (or `v6.0.0/preset-safe`) codemod, then you most proabably should not need to take any further action on these items. If there's a specific part of the breaking change that is not part of the codemod or needs some manual work, it will be listed in the end of each section.
```

Comment:
The sentence becomes a bit "clumsy" with those words included. It also makes it very uncertain from our point of view. 🤔 

```suggestion
If you have already applied the `v6.0.0/data-grid/preset-safe` (or `v6.0.0/preset-safe`) codemod, then you should not need to take any further action on these items. If there's a specific part of the breaking change that is not part of the codemod or needs some manual work, it will be listed in the end of each section.
```

Reply:
Yep, agreed. The next line should be enough to make the fact clear that some sub-portions may require some additional work.
I was trying to find a different emojie for such cases, something like a yellow check mark but couldn't find a suitable one 😄